src/google/protobuf/descriptor_database.cc

```diff
@@ -628,7 +628,7 @@ bool EncodedDescriptorDatabase::DescriptorIndex::AddFile(const FileProto& file,
                                                          Value value) {
   // We push `value` into the array first. This is important because the AddXXX
   // functions below will expect it to be there.
-  all_values_.push_back({value.first, value.second});
+  all_values_.push_back({value.first, value.second, nullptr});
```

Comment:
I haven't looked very closely but I don't think it is valid to pass nullptr here, since that will result in passing nullptr to the std::string constructor. Replacing nullptr with `{}` might work.

Reply:
updated